### ISSUE TITLE
Server start command

### ISSUE THREAD

**Issue by Zeromax** [pattern-lab/patternlab-php-core#10]
On windows this will not work: https://github.com/pattern-lab/patternlab-php-core/blob/dev/src/PatternLab/Console/Commands/ServerCommand.php#L53
```
configuring pattern lab...
server started on localhost:8080. use ctrl+c to exit...

Notice: Undefined index: _ in D:\www\patternlab\vendor\pattern-lab\core\src\PatternLab\Console\Commands\ServerCommand.php on line 44
Der Befehl "-S" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
```

This is cause `$_SERVER["_"]` is not set in windows CMD. I would prefer to set it up as an option `Config::get('phpBin')` so you can set it up as needed for each project.

**Comment by dmolsen** [comment]
I know this is terribly old but this has been addressed in `dev`. I'll close when I cut the next release.

**Comment by Zeromax** [comment]
Thanks.

**Comment by dmolsen** [comment]
Released with `v2.2.0`
